Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmon.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session Event ASIM parser for Sysmon (Event 3)
-  Version: '0.1'
-  LastUpdated: Jan 11, 2023
+  Version: '0.2.1'
```

Comment:
we don't have _Im_ parser. Need to add that too

Reply:
done